ci/tests.yaml

```diff
@@ -13,3 +13,6 @@ dependencies:
   - pytest-cov
   - setuptools_scm
   - pyuvdata
+  - pip
+  - pip:
+      - git+https://github.com/aelanman/lunarsky.git
```

Comment:
It would be good to get `lunarsky` onto pypi (and maybe conda forge) to make it easier for people to install. Doesn't have to be now, but we should update this yaml when there's a more canonical place to get it from.